cypress/integration/Switch/Displays_error.feature

```diff
@@ -0,0 +1,6 @@
+Feature: The Switch field displays an error when invalid
+
+    Scenario: The user submits a form with an unchecked required switch
```

Comment:
Here I'm also wondering how a single switch can be required.

Reply:
resolved in checkbox discussion